Docs: configure grafana database configuration MYSQL (#80939)
grafana/grafana: docs/sources/setup-grafana/configure-grafana/_index.md

### max_open_conn

> The maximum number of open connections to the database. For MYSQL, this setting needs to be configured on both Grafana and database sides [`sysvar_max_connections`](https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_max_connections.).

Reviewer: ```suggestion
The maximum number of open connections to the database. For MYSQL, configure this setting on both Grafana and the database. For more information, refer to [`sysvar_max_connections`](https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_max_connections).
```
